setup.py

```diff
@@ -55,6 +61,29 @@ def get_packages():
     return result
 
 
+class RefreshLangs(build_ext):
+    def run(self):
+        mo_files = []
+        prefix = 'keymapper'
+
+        for po_file in glob.glob(PO_FILES):
+            lang = splitext(basename(po_file))[0]
+            os.makedirs(join(lang, 'LC_MESSAGES'), exist_ok=True)
+            mo_path = join(lang, 'LC_MESSAGES', 'key-mapper.mo')
+            print('Generating translation for {}'.format(lang))
+            subprocess.run(['msgfmt', '-o', mo_path, str(po_file)], check=True)
+            mo_files.append(mo_path)
+
+        return mo_files
+
+lang_data = []
+for po_file in glob.glob(PO_FILES):
+    lang = splitext(basename(po_file))[0]
+    lang_data.append((
+        f'/usr/share/key-mapper/lang/{lang}/LC_MESSAGES',
+        [f'{lang}/LC_MESSAGES/key-mapper.mo']
```

Comment:
can you put all .mo files into a separate directory? `mo/` or something

Reply:
done